### ISSUE TITLE
Pub packages are not currently supported

### ISSUE THREAD

**Issue by canewsin** [ethanblake4/flutter_eval#18]
> Pub packages are not currently supported

Does this mean that we cannot reference pub packages at runtime right? I see that Flutter Gallery uses various pub packages, which is listed on `flutter_eval` readme.

**Comment by ethanblake4** [comment]
Yes, you cannot reference pub packages unless you create a bridge plugin for them.

Flutter gallery is listed as an example of how flutter_eval affects the size of the gallery app when flutter_eval is added to it. You are not able to compile/run flutter gallery inside of flutter_eval though yet.
